build/dependencies.props

```diff
@@ -1,6 +1,6 @@
-<Project>
+﻿<Project>
   <PropertyGroup>
-    <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
+    <AspNetIntegrationTestingVersion>0.4.0-*</AspNetIntegrationTestingVersion>
```

Comment:
This is a manually-versioned package for exactly this reason. It's never shipped.